### PR TITLE
Update default hyperparams and rescaled losses for galaxies benchmark

### LINKED ISSUE #15: Fail to replicate the graph task on single GPU for GNN
Hey, thanks for putting up the nice dataset!

I, however, didn't manage to replicate the learning curve of a GNN on a single GPU. I ran it as you say:
`python benchmarks/galaxies/train_cosmology.py
`

For me, the model converged to $0.3$ training loss, far from the reported $10^{-3}$. 
`Final checkpoint test loss [[0.21392891 0.39251247]].`

Also, the number of parameters printed was only half the size reported:
`Number of parameters: 700674`

Any ideas why that is the case? Thanks a lot!

### PR DESCRIPTION
**Pull request by julballa**
Should resolve #15

### CODE REVIEW

**Comment by smsharma** [comment]
Thanks for tracking this down!
